web-ng/src/app/components/project-header/project-header.component.html

```diff
@@ -13,16 +13,23 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-
+<img class="logo" src="/assets/img/logo.png" />
+<span class="ground"> Ground </span>
+<span> | </span>
 <app-inline-edit-title></app-inline-edit-title>
 <div *ngIf="auth.user$ | async as user; else login" class="rhs-widgets">
+  <button class="btn-share">
+    <mat-icon class="icon-share"> share </mat-icon>
```

Comment:
Is the text here the "alt text"? If so I assume the extra spaces are a stylistic choice?

Reply:
Removed the extra spaces, share is just a type of icon it renders